assets/js/theme/common/product-details.js

```diff
@@ -70,6 +70,7 @@ export default class ProductDetails {
                 }
             }
         } catch (e) {
+            /* eslint-disable no-console */
             console.error(e);
```

Comment:
Wouldn't it be more ideal to use `console.error(e) // eslint-disable-line no-console`? Just a preference on how we disable eslint modules for specific lines in projects on my end, so I could be absolutely wrong in how I'm using it.